[2201.7.0] - Add bal persist diagram to the overview page (#7147)
ballerina-platform/ballerina-dev-website: swan-lake/featured-scenarios/manage-data-persistence-with-bal-persist.md

This guide describes how to interact with the data store and perform operations against it using `bal persist`. This will cover the basic use case of creating, reading, updating, and deleting records on a data store in an organization. It also elaborates on how you can create an HTTP RESTful API using Ballerina that can be used to perform basic CRUD operations on the data store.

![Data Service Architecture](/learn/images/bal-persist-scenario-diagram.png "Data Service Architecture")

> >**Info:** This guide uses an in-memory data store for simplicity. However, the described methodology can also be applied to work with MySQL, MSSQL and, Google Sheets as the `bal persist` currently offers support for these three data stores: in-memory tables, MySQL and MSSQL databases, and Google Sheets.

Reviewer: ```suggestion
>**Info:** This guide uses an in-memory data store for simplicity. However, the described methodology can also be applied to work with MySQL, MSSQL, and Google Sheets, as the `bal persist` currently offers support for these three data stores: in-memory tables, MySQL and MSSQL databases, and Google Sheets.
```

> Note: All of the above auto-generated BAL files should not be modified.

> > Note: If you use other data stores like MySQL, MSSQL or Google Sheets, additional files are created to handle runtime configurations and scripts to set up the database/worksheet. Since `in-memory` is used, you don’t need any configuration and pre setting up to run the application.

Reviewer: ```suggestion
> Note: If you use other data stores like MySQL, MSSQL, or Google Sheets, additional files will get created to handle runtime configurations and scripts to set up the database/worksheet. Since `in-memory` is used, you don’t need to do any configuration or setting-up before running the application.
```
